sp800-63a/sec4_ial.md

```diff
@@ -154,90 +159,94 @@ The CSP SHOULD perform identity proofing in-person. The CSP MAY perform remote i
 
-### 4.6.4. Verification Requirements
+### 4.5.4. Verification Requirements
 
 See [Section 5.3, Identity Verification](#verify) for more information on acceptable identity evidence.
 
 - At a minimum, the applicant must be verified by a process that is able to achieve a strength of SUPERIOR.
 
-### 4.6.5. Presence Requirements
+### 4.5.5. Presence Requirements
 
 All identity proofing steps SHALL be performed in person. See [Section 5.3.3](#vip) for more details.
 
 Remote proofing SHALL NOT be allowed.
 
-### 4.6.6 Address Confirmation
+### 4.5.6 Address Confirmation
 
 - The CSP SHALL confirm address of record through validation of the address contained on any supplied, valid piece of identity evidence.
 - Self-asserted address data SHALL NOT be used for confirmation.
 - A notification of proofing SHALL be sent to the confirmed address of record.
 - The CSP MAY provide an enrollment code directly to the subscriber if binding to an authenticator will occur at a later time. The enrollment code SHALL be valid for a maximum of 7 days.
 
-### 4.6.7. Biometric Collection
+### 4.5.7. Biometric Collection
 
-The CSP SHALL collect and record a biometric sample at the time of proofing (e.g., facial image or fingerprints) the purposes of non-repudiation and re-proofing.  See [Section 5.2.3](#biometric_use) of SP 800-63B for more detail on biometric collection.
+The CSP SHALL collect and record a biometric sample at the time of proofing (e.g., facial image, fingerprints) the purposes of non-repudiation and re-proofing.  See [Section 5.2.3](#biometric_use) of SP 800-63B for more detail on biometric collection.
 
-### 4.6.8. Security  Controls
+### 4.5.8. Security  Controls
 
-The CSP SHOULD employ appropriately tailored security controls from the high baseline of security controls defined in [[SP 800-53]](#SP800-53) or an equivalent industry standard and SHOULD ensure that the minimum assurance requirements associated with the *high* baseline are satisfied.
+The CSP SHOULD employ appropriately tailored security controls from the High baseline of security controls defined in [[SP 800-53]](#SP800-53) or an equivalent industry standard and SHOULD ensure that the minimum assurance requirements associated with the *high* baseline are satisfied.
 
-## 4.7. Enrollment Code
+## 4.6. Enrollment Code
 An enrollment code allows the CSP to confirm that the applicant controls an address of record, as well as offers the applicant the ability to reestablish binding to their enrollment record.  Binding is not always completed in the same session as the original identity proofing transaction.  
 
 An enrollment code SHALL be comprised of one of the following:
 
-* Minimually, a random six (6) character alphanumeric.
-* A machine readable optical label, such as a QR Code, that contains data of similar or higher entropy as a random six (6) character alphanumeric.
+* Minimually, a random six character alphanumeric.
```

Comment:
@jimfenton i think we have something out on 737, but what if the proofing process just gave someone an authenticator, like a U2F device.  would the serial number work (I think so)?  Do we need to limit this to some sort of code that is off device?  Worth stating that?